### ISSUE TITLE
Modifying code for the F0 board

### ISSUE THREAD

**Issue by ChubbyBunny1** [mck1117/wideband#300]
Can anyone give some directions on how to modify and upload the code to the F0 board, what IDE to use, workflow. I have an ST-LINK V2 and CubeIDE installed  

Thank you

**Comment by mck1117** [comment]
Hi, what changes are you trying to make? If it's something that would be useful for others too, I'd be happy to implement them and get them in to official builds.

**Comment by ChubbyBunny1** [comment]
I am trying to make an analog output. I modified the PCB and added an opamp with RC filter at PB1 (TIM3_CH1) in order have outpt 0-5V analog. Since timer 3 is set up at high frequency it should be easy to add an extra PWM out right?

**Comment by rusefillc** [comment]
> I am trying to make an analog output. 
have you seen F103 boards doing this with a DAC?

**Comment by ChubbyBunny1** [comment]
> > I am trying to make an analog output.
> 
> have you seen F103 boards doing this with a DAC?

Yes ofcource but i want to make this board work with analog output too.
